actions/scripts/snap_jenkins.sh

```diff
@@ -162,47 +59,62 @@ function test_all_actions() # $1 = card, $2 = accel
 	# Get SNAP Action number from Card
 	MY_ACTION=`./software/tools/snap_maint -C $card -m 1`
 	for action in $MY_ACTION ; do
+		run_test=1;
 		case $action in
 		*"10140000")
-			test_hdl_example $card $accel
+			test_10140000 $card $accel
 			RC=$?
+			run_test=0
 		;;
-		*"10141000")
-			test_hls_memcopy $card $accel
-			RC=$?
+		*"10140001") # HDL NVMe example
+			cmd="./actions/hdl_nvme_example/tests/test_0x10140001.sh"
 		;;
-		*"10141001")
-			test_hls_sponge $card $accel
-			RC=$?
+		*"10141000") # HLS Memcopy
```

Comment:
Why not a for loop?

tests=100..123 100.... .. .. . .
for t in $tests ; do
   cmd=".......test_$t.sh
  ...
done

Might be simpler to enhance. We might even like to add some find actions -name "test*.sh" to figure out which tests are there.